Formula/ada.rb

```diff
@@ -0,0 +1,36 @@
+class Ada < Formula
```

Comment:
This should probably be called `ada-url` or `ada-parser` to avoid users mistaking it for the Ada compiler.

Reply:
I'll update the pull request after https://github.com/ada-url/ada/pull/341 is merged a new version is released. Thanks.

Reply:
I followed up on a new pull request: https://github.com/Homebrew/homebrew-core/pull/128939